commands/azureCommands/show-buildTask.ts

```diff
@@ -0,0 +1,39 @@
+import { Registry } from "azure-arm-containerregistry/lib/models";
+import { ResourceGroup } from "azure-arm-resource/lib/resource/models";
+import { Subscription } from "azure-arm-resource/lib/subscription/models";
+import { BuildTaskNode } from "../../explorer/models/taskNode";
+import * as acrTools from '../../utils/Azure/acrTools';
+import { AzureUtilityManager } from "../../utils/azureUtilityManager";
+import { quickPickACRRegistry, quickPickBuildTask, quickPickSubscription } from '../utils/quick-pick-azure';
+import { openTask } from "./task-utils/showTaskManager";
+
+export async function showBuildTaskProperties(context?: BuildTaskNode): Promise<any> {
+    let subscription: Subscription;
+    let registry: Registry;
+    let resourceGroup: ResourceGroup;
+    let buildTask: string;
+
+    if (context) { // Right click
+        subscription = context.susbscription;
+        registry = context.registry;
+        resourceGroup = await acrTools.getResourceGroup(registry, subscription);
+        buildTask = context.label;
```

Comment:
Don't use context.label, we could potentially show the tasks differently at some point, instead store the acquired task information in the nodes.